backend/src/main/java/corea/review/controller/ReviewController.java

```diff
@@ -0,0 +1,26 @@
+package corea.review.controller;
+
+import corea.auth.annotation.LoginMember;
+import corea.auth.domain.AuthInfo;
+import corea.review.dto.ReviewRequest;
+import corea.review.service.ReviewService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/review")
+@RequiredArgsConstructor
+public class ReviewController implements ReviewControllerSpecification{
+    private final ReviewService reviewService;
```

Comment:
[수정 요청] 선언 이후 스페이스바 + 개행 

```suggestion
@Controller
@RequestMapping("/review")
@RequiredArgsConstructor
public class ReviewController implements ReviewControllerSpecification {
    
    private final ReviewService reviewService;
```